src/version/features/UserTour/content.js

```diff
@@ -81,7 +85,7 @@ const lastStep = {
     content: (
         <div className="tour-body">
             <p className="tour-body-content">
-                For more information and a guide on how to use {homepage.title}, please go to <a className="link" href={homepage.link} title={homepage.title} style={{color:"#3596f4", 'text-decoration': "none" }} target="_blank">
+                For more information and a guide on how to use {homepage.title}, please go to <a className="link" href={homepage.link} title={homepage.title} style={{color: themeColor, 'text-decoration': "none" }} target="_blank">
```

Comment:
use material-ui's CSS in JS instead of styles

Reply:
Done